app/command.go

```diff
@@ -155,10 +156,17 @@ func (a *App) ListAllCommands(teamId string, T goi18n.TranslateFunc) ([]*model.C
 }
 
 func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	parts := strings.Split(args.Command, " ")
-	trigger := parts[0][1:]
+	trigger := ""
+	message := ""
+	index := strings.IndexFunc(args.Command, unicode.IsSpace)
+	if index != -1 {
+		trigger = args.Command[:index]
+		message = args.Command[index+1:]
+	} else {
+		trigger = args.Command
+	}
+	trigger = trigger[1:] // remove '/'
```

Comment:
Would it make sense to check for `/` explicitly? I bet this method behaves unexpectedly if a `CommandArgs` is provided without a leading slash.

Reply:
yeah. I am not sure what inputs to expect. if it gets sanitized, or how this feature works end to end. or how it gets to this function ```executeCommand```   I will look into it.  

I think checking explicitly would be good. how should we treat an error? throw an exception?

Reply:
I think an error probably suffices to handle this case, yes.